tests/Integration/Behaviour/Features/Scenario/Product/Shop/delete_image_multishop.feature

```diff
@@ -26,10 +26,10 @@ Feature: Delete product image from Back Office (BO)
       | type        | standard       |
     And product "product1" type should be standard
     And product "product1" should have no images
-    And I copy product product1 from shop shop1 to shop shop2
     And I add new image "image1" named "app_icon.png" to product "product1" for shop "shop1"
-    And I copy product product1 from shop shop1 to shop shop3
-    And I copy product product1 from shop shop1 to shop shop2
```

Comment:
Actually, while I was starting to review the behat test I started thinking about this kind of scenarios.

I'm afraid the global set command will have less possibilities than the independent Copy/Delete had, in some scenarios we purposefully did stuff like:
- add image1 to shop1
- copy shop1 to shop2
- add image2 to shop2
- copy shop2 to shop3

This allowed to have different images between the shops, now that the set/copy/delete is done by a single command at once is it still doable? Since the handler only copies to shops that are not associated yet it should be possible though, by calling the set command with more and more shopIds.

One use case that is not possible anymore though, is in case we want to force the copy towards an already associated shop, but it's not a use case e need in the UX for now and it was never tested this way anyway. So nevermind it should all be fine.